payloads/library/GitBunnyGit/payload.txt

```diff
@@ -18,19 +18,20 @@ LED R G B
 ATTACKMODE RNDIS_ETHERNET 
 #ATTACKMODE ECM_ETHERNET
 
-source bunny_helpers.sh
-
 # Set your desired repo url and branch if you're not looking to update from mainline
 git_repo="https://github.com/hak5/bashbunny-payloads.git"
 git_branch="master"
 payloads_dir="/root/udisk"
-log_file="/root/udisk/payloads/$SWITCH_POSITION/git.log"
+log_file="/var/log/git.log"
+
+echo " ---------  Git Bunny Git [started] -----------" >> $log_file
 
-echo "Git Bunny Git" > $log_file
+# Sanity check on mounted drive
+[[ ! `mount | grep "nandf"` ]] && { LED R 200; echo "Could not mount filesystem" >> $log_file; exit 1; }
```

Comment:
I am a web dev by profession... I am constantly error handling cases that should never happen, but seem to happen.